packages/atomic/cypress/integration/common-assertions.ts

```diff
@@ -81,6 +81,20 @@ export function assertConsoleErrorMessage(msg: string) {
   });
 }
 
+export function assertConsoleWarning(warn = true) {
```

Comment:
I know you did not cause those linter errors above, but could you fix them at the same time as touching this file ? 😄

Reply:
Sure!